src/sw/to-cache.ts

```diff
@@ -13,92 +13,83 @@ function subtractSets<T extends any>(set1: Set<T>, set2: Set<T>): Set<T> {
-  initalJs,
-  new Set([initialApp.main, ...initialApp.deps]),
-);
-
-export const initial = [
-  '/',
-  ...initalJs,
-  logo,
-  githubLogo,
-  largePhotoIcon,
-  artworkIcon,
-  deviceScreenIcon,
-  logoIcon,
-  logoWithText,
-];
 
 // The processors and codecs
 // Simple stuff everyone gets:
 import * as featuresWorker from 'entry-data:../features-worker';
-import rotateWasm from 'url:codecs/rotate/rotate.wasm';
-import quantWasm from 'url:codecs/imagequant/imagequant.wasm';
-import resizeWasm from 'url:codecs/resize/pkg/squoosh_resize_bg.wasm';
-import hqxWasm from 'url:codecs/hqx/pkg/squooshhqx_bg.wasm';
-import mozjpegWasm from 'url:codecs/mozjpeg/enc/mozjpeg_enc.wasm';
 
 // Decoders (some are feature detected)
 import * as avifDec from 'entry-data:codecs/avif/dec/avif_dec';
-import avifDecWasm from 'url:codecs/avif/dec/avif_dec.wasm';
-import jxlDecWasm from 'url:codecs/jxl/dec/jxl_dec.wasm';
 import * as webpDec from 'entry-data:codecs/webp/dec/webp_dec';
-import webpDecWasm from 'url:codecs/webp/dec/webp_dec.wasm';
-import wp2DecWasm from 'url:codecs/wp2/dec/wp2_dec.wasm';
 
 // AVIF
-import * as avifEncMtWorker from 'entry-data:codecs/avif/enc/avif_enc_mt.worker.js';
 import * as avifEncMt from 'entry-data:codecs/avif/enc/avif_enc_mt';
-import avifEncMtWasm from 'url:codecs/avif/enc/avif_enc_mt.wasm';
-import avifEncWasm from 'url:codecs/avif/enc/avif_enc.wasm';
-import * as avifEnc from 'entry-data:codecs/avif/enc/avif_enc.js';
+import * as avifEnc from 'entry-data:codecs/avif/enc/avif_enc';
 
 // JXL
-import * as jxlEncMtWorker from 'entry-data:codecs/jxl/enc/jxl_enc_mt.worker.js';
+import * as jxlEncMtSimd from 'entry-data:codecs/jxl/enc/jxl_enc_mt_simd';
 import * as jxlEncMt from 'entry-data:codecs/jxl/enc/jxl_enc_mt';
-import jxlEncMtWasm from 'url:codecs/jxl/enc/jxl_enc_mt.wasm';
-import jxlEncWasm from 'url:codecs/jxl/enc/jxl_enc.wasm';
 import * as jxlEnc from 'entry-data:codecs/jxl/enc/jxl_enc';
 
 // OXI
-import oxiMtWasm from 'url:codecs/oxipng/pkg-parallel/squoosh_oxipng_bg.wasm';
-import oxiWasm from 'url:codecs/oxipng/pkg/squoosh_oxipng_bg.wasm';
+import * as oxiMt from 'entry-data:codecs/oxipng/pkg-parallel/squoosh_oxipng';
+import * as oxi from 'entry-data:codecs/oxipng/pkg/squoosh_oxipng';
 
 // WebP
+import * as webpEncSimd from 'entry-data:codecs/webp/enc/webp_enc_simd';
 import * as webpEnc from 'entry-data:codecs/webp/enc/webp_enc';
-import webpEncWasm from 'url:codecs/webp/enc/webp_enc.wasm';
 
 // WP2
-import * as wp2EncMtWorker from 'entry-data:codecs/wp2/enc/wp2_enc_mt.worker.js';
+import * as wp2EncMtSimd from 'entry-data:codecs/wp2/enc/wp2_enc_mt_simd';
 import * as wp2EncMt from 'entry-data:codecs/wp2/enc/wp2_enc_mt';
-import wp2EncMtWasm from 'url:codecs/wp2/enc/wp2_enc_mt.wasm';
 import * as wp2Enc from 'entry-data:codecs/wp2/enc/wp2_enc';
-import wp2EncWasm from 'url:codecs/wp2/enc/wp2_enc.wasm';
+
+export function shouldCacheDynamically(url: string) {
+  return url.startsWith('/c/demo-');
```

Comment:
I'm a bit worried about this. Our old build system behaved like this and it caught us out - things started caching differently after a file rename, and that wasn't expected. I deliberately changed it to a system where you'd get a failure if you renamed a file but didn't also change this file, rather than a behaviour change creeping in.

Reply:
There was already this `.startsWith('/c/demo-')` in the code, I only extracted it into a function to avoid hardcoding in two places instead of one.

Reply:
Hah, then it was I that did a bad job. I should fix that at some point, but no worried for now.

Reply:
Check src/sw/index.js change: https://github.com/GoogleChromeLabs/squoosh/pull/1027/files#diff-46250d1a173a5036cad883e5b6161d0e996c5a669cccc797034e5e776dbd4525